packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -9,6 +9,13 @@ it('should render a default radio field', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render a default radio field without crashing if it has an undefined child', () => {
```

Comment:
nit: lets remove without crashing. all of these tests shouldn't crash the component 😃

Reply:
Removed: 56b2ee965

Reply:
There is also a potential for crashing if the child does not have the type property.

Reply:
@danplubell K I will cover that as well.

Reply:
@danplubell Just curious, is that possible?

Reply:
Yes, if the child is not a Radio element or anything else that might not have the type property.  For instance, there isn't anything to prevent someone from having a mix of Radio elements and Select elements as children of the RadioField.

Reply:
Added: 0f1b14558

Reply:
@danplubell & @yfaney I discussed this with the terra team and we're ok allowing non-radio elements in radio form field. Namely the case we discussed was adding some kind of image next to a radio button. Radio form field will add screen reader accessibility to direct radio children, any non radio children will need to be evaluated for accessibility. We don't recommend adding any other form elements to the radio field but should not prevent it.

Reply:
@mjhenkes @danplubell Reverted: 192dab011